client/src/pages/create_game/CreateGame.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import Input from "@material-ui/core/Input";
+import Typography from "@material-ui/core/Typography";
+
+import Header from "../common/Header";
+import "./CreateGame.css";
+
+function CreateGame() {
+  return (
+    <>
```

Comment:
curious - any reason for an extra element wrapper here?

Reply:
My apologies. I had 2 separate elements that needed to be wrapped before but not anymore. Thanks for pointing it out!